Detectors/TRD/reconstruction/macros/plotAllCharges.C

```diff
@@ -0,0 +1,96 @@
+#if !defined(__CLING__) || defined(__ROOTCLING__)
+// ROOT header
+#include <TROOT.h>
+#include <TChain.h>
+#include <TH2.h>
+#include <TF1.h>
+#include <TCanvas.h>
+#include <TStyle.h>
+#include <TProfile.h>
+#include <TFile.h>
+#include <TTree.h>
+#include <TAxis.h>
+#include <TLine.h>
+// O2 header
+#include "DataFormatsTRD/TriggerRecord.h"
+#include "DataFormatsTRD/Digit.h"
+#include "DataFormatsTRD/Tracklet64.h"
+#include "DataFormatsTRD/Constants.h"
+#endif
+
+using namespace o2::trd;
+
+std::vector<TH1F*> createTrdChargeHists()
+{
+  std::vector<TH1F*> hCharges;
+  for (int sector = 0; sector < 18; ++sector) {
+    hCharges.push_back(new TH1F(Form("Charge0_%i", sector), Form("Sector%i hCharge window 0;charge;counts", sector), 64, 0, 63));
+    hCharges.push_back(new TH1F(Form("Charge1_%i", sector), Form("Sector%i Charge window 1;charge;counts", sector), 256, 0, 255));
+    hCharges.push_back(new TH1F(Form("Charge2_%i", sector), Form("Sector%i Charge window 2;charge;counts", sector), 256, 0, 255));
+  }
+  return hCharges;
+}
+
+void plotAllCharges()
```

Comment:
Did you upload the correct macros? Basically this does the same as `checkTrackletCharges.C`, right?

Reply:
I removed that macro, its pointless, you can see anything.